environment.yml

```diff
@@ -31,8 +31,8 @@ dependencies:
         - mujoco-py<1.50.2,>=1.50.1
         - cached_property
         - cloudpickle
-        - git+https://github.com/Lasagne/Lasagne.git@7992faa80fa5233a786e2582a605e854cea7d1cf#egg=Lasagne
-        - git+https://github.com/Theano/Theano.git@2c19431b165aaf8289d1e6654988c64f4bb234e1#egg=Theano
+        - git+https://github.com/Theano/Theano.git@adfe319ce6b781083d8dc3200fb4481b00853791#egg=Theano
+        - git+https://github.com/neocxi/Lasagne.git@484866cf8b38d878e92d521be445968531646bb8#egg=Lasagne
```

Comment:
These are the two main lines that fix all the tests. Someone let me know if this will break any code.

Reply:
Does it not work with the latest version of theano (but the modified lasagne)?

Reply:
Unfortunately no. And I'm honestly not sure why that is